tasks/go.py

```diff
@@ -398,13 +398,27 @@ def check_mod_tidy(ctx, test_folder="testmodule"):
 
         if errors_found:
             message = "\nErrors found:\n" + "\n".join("  - " + error for error in errors_found)
-            message += "\n\nRun 'inv tidy-all' to fix 'out of sync' errors."
+            message += "\n\nRun 'inv tidy' to fix 'out of sync' errors."
             raise Exit(message=message)
 
 
 @task
 def tidy_all(ctx):
-    for mod in DEFAULT_MODULES.values():
+    sys.stderr.write(color_message('This command is deprecated, please use `tidy` instead\n', "orange"))
+    sys.stderr.write("Running `tidy`...\n")
+    tidy(ctx)
+
+
+@task
+def tidy(ctx, only_modified_packages=False):
+    # TODO: if only_modified_packages then use `go mod tidy -diff` with golang 1.23, probably in a separate check-mod-tidy task
+    # https://github.com/golang/go/issues/27005
+    from tasks import get_modified_packages
+
+    # TODO: Also include packages that import them
```

Comment:
I think to do that you could use `get_impacted_packages` that is used in the CI it returns all the modified packages and the one that import them recursively